_docs/style/colors.md

```diff
@@ -60,32 +107,144 @@ illustrations. They should never overpower the primary colors.
         {%endfor%}
       </div>
     {%endfor%}
-  </div>
+</div>
+{% endcolumn %}
+
+</div>
+
 </section>
 
+<section class="static-section" markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+
+You can also use status labels as a way to communicate the status of an object, for example, a support ticket. Unlike health status which is a closed system that prescribes that you use specific icons and colors to convey meaning, status labels are non-use case specific, and you can use them as you see fit. Review your system of statuses to determine which label to use.
```

Comment:
The non-health status family offers more freedom than the Health Status family. 
You can choose from among the family colors to attract the right amount of attention to the status of an object.

Reply:
Done.